psi4/src/psi4/dct/dct.h

```diff
@@ -320,9 +338,9 @@ class DCTSolver : public Wavefunction {
     /// The maximum number of IDPs possible for cumulant updates
     int dim_cumulant_;
     /// The lookup array that determines which compound indices belong to orbital IDPs and which don't
-    int *lookup_orbitals_;
+    int* lookup_orbitals_;
```

Comment:
Any reason these int* couldn't be a std::vector?

Reply:
Good call! `std::vector<bool>` is a more sensible type, since the `int` is only ever 0 or 1.